src/objcopy.zig

```diff
@@ -442,7 +442,7 @@ const BinaryElfOutput = struct {
     }
 
     fn sectionValidForOutput(shdr: anytype) bool {
-        return shdr.sh_size > 0 and shdr.sh_type != elf.SHT_NOBITS and
+        return shdr.sh_size >= 0 and shdr.sh_type != elf.SHT_NOBITS and
```

Comment:
I believe this value is unsigned, so please simply delete the comparison:

```suggestion
        return shdr.sh_type != elf.SHT_NOBITS and
```